.editorconfig

```diff
@@ -0,0 +1,17 @@
+# editorconfig.org
+root = true
+
+[*]
```

Comment:
Should we force a default style for files not matching any extension? Not sure if it is a good idea to over-force in this case as the potential files can be quite different.

Could be the `[*]` section safely removed?

Reply:
```

Reply:
Suggested fix: e55fdb6

Reply:
> The only controversial one is `indent_style = tab` - maybe should probably be redefined for `Dockerfile` only?

Yes, that's controversial... especially when other file formats in the repo are using `indent_style = space` :). Is there any syntactical requirement that enforce tabs in Dockefiles? Or it's a matter of taste decide between spaces and tabs?

In the second case, I'd suggest to let Dockerfile out of the scope of this file.

> There are a couple of `*.sh` bash scripts in the repo - which are inconsistently spaced - should they be added?

Style & homogeneity for JavaScript source code is great. Style & homogeneity for JSON files could be also great. However, going to far trying to cover little file of the repository could be excessive... I'd suggest by the moment leaving out *.sh files.

Reply:
So something like this: caaffcb 

- Default to Unix line endings and UTF-8
- agnostic over tabs/spaces. 
- Specific  config for  JS, md, yaml etc for the things that the linter cares about.

Note that editorConfig is just a suggestion. Husky should run the prettifier to re-enforce code style.

Reply:
After the last changes the .eseditorcnofig file looks nice :) Thx!

NTC